Makefile

```diff
@@ -86,8 +89,13 @@ bin/unpack:
 build-local-container: ## Builds provisioner container image locally
 	docker build -f Dockerfile -t $(IMAGE) .
 
-load-image: ## Load-image loads the currently constructed image onto the cluster
-	 kind load docker-image $(IMAGE)
+kind-load: ## Load-image loads the currently constructed image onto the cluster
```

Comment:
looks like if we have a custom kind cluster name then this command needs to accept the name as a flag to load successfully, ie` kind load docker-image $(IMAGE) --name $(KIND_CLUSTER_NAME)` otherwise we see https://github.com/operator-framework/rukpak/runs/5386209881?check_suite_focus=true